controllers/slice/reconciler.go

```diff
@@ -386,7 +386,7 @@ func isAppPodStatusChanged(current []kubeslicev1beta1.AppPod, old []kubeslicev1b
 // Setup SliceReconciler
 // Initializes metrics and sets up with manager
 func (r *SliceReconciler) Setup(mgr ctrl.Manager, mf metrics.MetricsFactory) error {
-	gaugeAppPods := mf.NewGauge("app_pods", "App pods in slice", []string{"slice", "namespace"})
+	gaugeAppPods := mf.NewGauge("app_pods", "App pods in slice", []string{"slice", "slice_namespace"})
```

Comment:
What confusion is here btw? @kmjayadeep

Reply:
prometheus adds a few labels by default, which includes `pod` name as the operator pod name (source of the metric). So it looks like this namespace is the namespace where the operator runs.